docs/tutorials/schedule.md

```diff
@@ -0,0 +1,304 @@
+    policy='cyclic',
+    target_ratio=(10, 1e-4),
+    cyclic_times=1,
+    step_ratio_up=0.4,
+)
+momentum_config = dict(
+    policy='cyclic',
+    target_ratio=(0.85 / 0.95, 1),
+    cyclic_times=1,
+    step_ratio_up=0.4,
+)
+```
+
+**After completing your configuration file，you could use [learning rate visualization tool](https://mmclassification.readthedocs.io/zh_CN/latest/tools/visualization.html#id3) to draw the corresponding learning rate adjustment curve.**
+
+## Use multiple learning rates and weight_decays
+
+Some models may have some parameter-specific settings for optimization, for example, no weight decay to the BatchNorm layer or using different learning rates for different network layers.
+MMClassification provides `paramwise_cfg` for configuration, please refer to [MMCV](https://mmcv.readthedocs.io/en/latest/_modules/mmcv/runner/optimizer/default_constructor.html#DefaultOptimizerConstructor).
+
+- Using specified options
+
+    MMClassification provides options including `bias_lr_mult`, `bias_decay_mult`, `norm_decay_mult`, `dwconv_decay_mult`, `dcn_offset_lr_mult` and `bypass_duplicate` to specify all relevant `bais`, `conv and`, `dw bypass` parameter. E.g:
+
+    1. No weight decay to the BatchNorm layer
+
+    ```python
+    paramwise_cfg = dict(norm_decay_mult=0.)
+    ```
+
+- Using `custom_keys` dict
+
+    MMClassification can use `custom_keys` to specify different parameters to use different learning rates or weight decays,
+    for example:
+
+    1. No weight decay for specific parameters
+
+    ```python
+    paramwise_cfg = dict(
+        custom_keys={
+            '.backbone.cls_token': dict(decay_mult=0.0),
+            '.backbone.pos_embed': dict(decay_mult=0.0)
+        })
+    ```
+
+    2. Using a smaller learning rate and aweight decay for the backbone layers
+
+    ```python
+    paramwise_cfg = dict(custom_keys={'.backbone': dict(lr_mult=0.1, decay_mult=0.9)})s
+    # backbone 的 'lr' and 'weight_decay' 分别为 0.1 * lr 和 0.9 * weight_decay
+    ```
+
+## Gradient clipping and gradient accumulation
+
+Based on the PyTorch basic optimizer, MMCV enhances the optimizer's functions, such as gradient clipping, gradient accumulation, etc., refer to [MMCV](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/optimizer.py).
+
+### Gradient clipping
+
+During the training process, abnormal points may cause some model gradients to explode. Gradient clipping needs to be used to stabilize the training process.
+Currently `clip_grad_norm_` is supported, please refer to [PyTorch Documentation](https://pytorch.org/docs/stable/generated/torch.nn.utils.clip_grad_norm_.html).
+Examples are as follows:
+
+```python
+optimizer_config = dict(grad_clip=dict(max_norm=35, norm_type=2))
+```
+
+When the optimizer hook type is not specified, `OptimizerHook` is used by default, and the above is equivalent to:
+
+```python
+optimizer_config = dict(type="OptimizerHook", grad_clip=dict(max_norm=35, norm_type=2))
```

Comment:
This section is not relative to gradient clipping, move it to "Gradient accumulation"